test/example/tutorials/test_decompositions.py

```diff
@@ -0,0 +1,221 @@
+import math
+
+import sympy as sp
+from IPython.display import display
+
+from opensquirrel.circuit_builder import CircuitBuilder
+from opensquirrel.decomposer.aba_decomposer import XZXDecomposer, ZYZDecomposer
+from opensquirrel.decomposer.cnot_decomposer import CNOTDecomposer
+from opensquirrel.decomposer.mckay_decomposer import McKayDecomposer
+from opensquirrel.default_gates import H, Rx, Rz
+from opensquirrel.ir import Float, Qubit
+
+
+class TestDecomposition:
+    def test_sympy_disp(self):
```

Comment:
What is `disp`? I would change the name into  `test_simpy_<whatever full word>`.

Reply:
```suggestion
    def test_sympy_output(self):
```

Reply:
Why would we test sympy code?